app/channels.go

```diff
@@ -31,6 +32,22 @@ type configSvc interface {
 	SaveConfig(newCfg *model.Config, sendConfigChangeClusterMessage bool) (*model.Config, *model.Config, *model.AppError)
 }
 
+type licenseSvc interface { // nolint: unused
```

Comment:
I am not seeing this being used in the Channels struct. All licensing calls seem to be attached to server. Wondering if we still need all these methods in the interface?

Looking at the plugin API, we only have `GetLicense` and `RequestTrialLicense`. If there are other licensing related calls in boards and playbooks, we should only add those. Our immediate objective should be to bring boards and playbooks together in the same binary.